### PR TITLE
ios: fix group member sheet load animation

### PR DESCRIPTION
**Pull request by levitatingpineapple**

### CODE REVIEW

File: apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift [hunk 1]
```diff
@@ -253,6 +153,114 @@ struct GroupMemberInfoView: View {
         }
         .modifier(ThemedBackground(grouped: true))
     }
+    
+    @ViewBuilder
+    private func sections(_ member: GroupMember) -> some View {
```
Comment (levitatingpineapple) [inline apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift:158]: moved without changes

Comment (epoberezkin) [reply apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift:158]: why does it need to move?

File: apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift [hunk 2]
```diff
@@ -94,129 +95,28 @@ struct GroupMemberInfoView: View {
...
                     // this condition prevents re-setting picker
                     if !justOpened { return }
                 }
                 justOpened = false
-                DispatchQueue.main.async {
-                    newRole = member.memberRole
-                    do {
-                        let (_, stats) = try apiGroupMemberInfo(groupInfo.apiId, member.groupMemberId)
-                        let (mem, code) = member.memberActive ? try apiGetGroupMemberCode(groupInfo.apiId, member.groupMemberId) : (member, nil)
+                newRole = member.memberRole
```
Comment (epoberezkin) [inline apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift:107]: this is state change on background thread?

File: apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift (hunk 1 above)
Comment (epoberezkin) [reply apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift:160]: without moving, it weirdly works differently...

**Pull request merged**